dockertest/docker_daemon.py

```diff
@@ -153,14 +153,22 @@ def restart():
     return _systemd_action('restart')
 
 
+def systemd_show(prop):
+    """
+    Runs 'systemctl show --property=ARG' for given arg.
+    Returns the value as a string.
+    """
+    cmd_result = _systemd_action('show --property=%s' % prop)
+    stdout = cmd_result.stdout
+    if not stdout.startswith(prop + '='):
+        raise RuntimeError("Unexpected output from %s: expected %s=XXXX,"
+                           " got '%s'" % (cmd_result.command, prop, stdout))
+    return stdout[(len(prop) + 1):].strip()
+
+
```

Comment:
I assume you mean systemd_*show*, not _action. Good thinking. It'll take me a while because python testing always takes me a while; I'll have something ready tomorrow.